VotingApplication/VotingApplication.Web/Api/Models/DBViewModels/PollRequestResponseModel.cs

```diff
@@ -7,6 +7,7 @@ namespace VotingApplication.Web.Api.Models.DBViewModels
     public class PollRequestResponseModel
     {
         public Guid UUID { get; set; }
+        public Guid ManageId { get; set; }
```

Comment:
Good point. It was necessary when api/get was returning a PollRequestResponseModel, but it now returns a DashboardPollResponseModel, so this change should have been reverted.